Revit_Core_Engine/Convert/Environment/FromRevit/Pipe.cs

```diff
@@ -41,7 +40,7 @@ public static partial class Convert
         [Input("revitPipe", "Revit pipe to be converted.")]
         [Input("settings", "Revit settings.")]
         [Input("refObjects", "Referenced objects.")]
```

Comment:
I would consider rewriting these 2 descriptions - they will reappear in multiple places so might be worth. What about:

```suggestion
        [Input("settings", "Revit adapter settings.")]
        [Input("refObjects", "A collection of objects processed in the current adapter action, stored to avoid processing the same object more than once.")]
```

Reply:
All instances updated